src/lib.rs

```diff
@@ -258,6 +306,8 @@ pub struct Sgp30<I2C, D> {
     initialized: bool,
 }
 
+const SELFTEST_SUCCESS: &[u8] = &[0xd4, 0x00];
```

Comment:
Could you add a rustdoc comment explaining this magic number? (Previously it was obvious because it was part of the "selftest" command.) Maybe something like this?

```suggestion
/// The fixed data pattern returned when the on-chip self-test is successful.
const SELFTEST_SUCCESS: &[u8] = &[0xd4, 0x00];
```

Reply:
Sure, that seems good to me!